frontend/src/main/web/src/components/Modal/index.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { PropTypes }from 'react'
+import { Modal as OverlayModal } from 'react-overlays'
+import { merge } from 'lodash'
+import Header from './ModalHeader'
+import Title from './ModalTitle'
+import Body from './ModalBody'
+import Footer from './ModalFooter'
```

Comment:
No reason to use different names for these. Call them `ModalHeader` etc. to keep things more consistent.